scout/parse/omim.py

```diff
@@ -349,7 +349,7 @@ def get_mim_genes(genemap_lines, mim2gene_lines):
     return hgnc_genes
 
 
-def get_mim_phenotypes(genemap_lines: Iterable[str]) -> Dict[int, Any]:
+def get_mim_phenotypes(genemap_lines: Iterable[str]) -> dict[str, Any]:
```

Comment:
I think this change is breaking the function:

```
(py38) chiararasi@n187-p80 scout % scout --demo update diseases -f /Users/chiararasi/Documents/work/scout-update
Traceback (most recent call last):
  File "/Users/chiararasi/miniconda3/envs/py38/bin/scout", line 33, in <module>
    sys.exit(load_entry_point('scout-browser', 'console_scripts', 'scout')())
  File "/Users/chiararasi/miniconda3/envs/py38/bin/scout", line 25, in importlib_load_entry_point
    return next(matches).load()
  File "/Users/chiararasi/miniconda3/envs/py38/lib/python3.8/importlib/metadata.py", line 77, in load
    module = import_module(match.group('module'))
  File "/Users/chiararasi/miniconda3/envs/py38/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/Users/chiararasi/Documents/work/GITs/scout/scout/commands/__init__.py", line 1, in <module>
    from .base import cli
  File "/Users/chiararasi/Documents/work/GITs/scout/scout/commands/base.py", line 12, in <module>
    from scout.commands.convert import convert
  File "/Users/chiararasi/Documents/work/GITs/scout/scout/commands/convert.py", line 7, in <module>
    from scout.parse.panel import parse_genes
  File "/Users/chiararasi/Documents/work/GITs/scout/scout/parse/panel.py", line 15, in <module>
    from .omim import get_mim_genes
  File "/Users/chiararasi/Documents/work/GITs/scout/scout/parse/omim.py", line 352, in <module>
    def get_mim_phenotypes(genemap_lines: Iterable[str]) -> dict[str, Any]:
TypeError: 'type' object is not subscriptable
(py38) chiararasi@n187-p80 scout %
```

Reply:
This could be due to the fact that I'm using Python 3.8. Which Python have you installed in your conda env, if you have Python >=3.9 than it makes sense that for you is working as you have written (type hinting generics in standard collections starts from 3.9 -> https://docs.python.org/3/whatsnew/3.9.html). I think you should revert it since we should support 3.8 until it gets deprecated.